core/src/main/java/nl/nn/adapterframework/webcontrol/api/CXFAuthorizationFilter.java

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2016-2017 Integration Partners B.V.
```

Comment:
Make sure the copyright notice is correct and up to date :)
If you work on a class you need to add the current year to it.